code/modules/reagents/chemistry_reagents/other.dm

```diff
@@ -34,8 +34,24 @@
 	description = "A ubiquitous chemical substance that is composed of hydrogen and oxygen."
 	reagent_state = LIQUID
 	color = "#0064C8" // rgb: 0, 100, 200
-	custom_metabolism = REAGENTS_METABOLISM * 0.05
-	taste_description = "water"
+	overdose_threshold = REAGENTS_OVERDOSE //I want the 30% chance of the "nauseous message
+	overdose_crit_threshold = REAGENTS_OVERDOSE_CRITICAL * 2 //100u
+	taste_description = "hydration"
+
+/datum/reagent/water/on_mob_life(mob/living/L, metabolism)
+	L.reagents.remove_all_type(/datum/reagent, REM, 0, 1) //slowly washes out chems in you
+	if(volume < 4)
+		L.adjustStaminaLoss(-10*REM)
+		L.adjustBruteLoss(-0.2, 0)
+		L.adjustFireLoss(-0.2, 0)
+	if(volume < 30)
+		L.adjustStaminaLoss(-2*REM)
```

Comment:
This benefit should only happen if there's a trade-off.

Reply:
It purges all reagants, including medicine at 1u/tick. ~~including itself, so it's a .5u/tick purge rate~~ If you're drinking water to get the 1u/stamina/tick, you're wasting medicine.

Reply:
Water is becoming highly unrealistic. It's the new charcoal/hypervene, that gives stamina.
Give it much milder effects, please.

Reply:
It's about as mild as it can get and have an effect, but okay...

New water plan-
Under 4u, it grants 2 stamina/tick and heals .1 brute/burn.
Under 30u, it grants .5 stamina/tick.
Over 5u, it causes a flat 0.1 slowdown.

Reply:
Changed, everything added but the slowdown, I don't want to touch movespeed with this PR anymore.

Reply:
water op?